### PR TITLE
Css flexible

### PR DESCRIPTION
**Pull request by ssahara**
The definitionlist plugin does not set any class name to the dl element, and style rules defined in the style.css in the plugin directory may affect rules with my own rules in the conf/userstyle.css for dl, dt, dd elements. This makes me difficult to define user own rules for the definition list. This request contains three commits, of two are to increase css flexibility of the definitionlist plugin:
- plugin config variable "stylename" for dl element
- use plugin config variable "dt_fancy" instead of DL_FANCY

The admin can set appropriate value through the Configuration manager. Language files prepared for English and Japanese. This request also contains removal of old function getInfo() replaced by plugin.info.txt file since DokuWIki 2009-12-25 Lemming.

Thanks for your setup gitHub repository of the definitionlist plugin for better source accessibility.

Satoshi

### CODE REVIEW

File: syntax.php [hunk 1]
```diff
@@ -37,11 +37,19 @@
 if (!defined('DL_DT')) define('DL_DT', ';');     // character to indicate a term (dt)
 if (!defined('DL_DD')) define('DL_DD', ':');     // character to indicate a definition (dd)
 
-// define the html used to generate the definition list
-// - set to false or 0 to use simple list html <dl><dt>term</dt><dd>definition</dd> ... </dl>
-// - set to true or 1 to use wrap the term element in a span permitting more complex styling
-//   <dl><dt><span class='term'>term</span></dt><dd>definition</dd> ... </dl>
-if (!defined('DL_FANCY')) define('DL_FANCY', true);
+// ---------- [ Optional config parameters ] ------------------------
```
Comment (samwilson) [inline syntax.php:40]: I know the previous author has done it differently, but really all comments like this should be included in the class- or file-level docblocks.

**Comment by samwilson** [comment]
Thanks for working on this! Looks good.

Just a couple of stylistic changes (see above) and I'll merge it.

Have you contacted the original author at all, by the way. Perhaps we should adopt this plugin? At the moment this is just an informal fork.

**Comment by samwilson** [comment]
Sorry, disregard my comment (now deleted) about getInfo(). I'd forgotten that it's no longer required. :-)

**Comment by samwilson** [comment]
I've merged this, thanks!

I changed the default for dt_fancy to true, to match the current behaviour, and extended the tests to cover your new feature. Forgive me if I've gone amiss in any way! :-)

Thanks again.

**Comment by ssahara** [comment]
Hi, SamWilson.

Thanks for merging my request. The programming is my hobby in weekend and late evening. Sorry for late reply.
I understand this repo is unformal fork. I will sometime contact the original author to inform your repo and ask if he intend to setup his official repository for definition plugin. I will send same email to you at the time. Is this ok for you?

**Pull request merged**